build.gradle.kts

```diff
@@ -1,18 +1,19 @@
-plugins {
-    `kotlin-dsl`
-}
-
 buildscript {
+
+    extra["targetSDKVersion"] = 33
+    extra["compileSDKVersion"] = 33
+    extra["minSDKVersion"] = 23
+
     repositories {
         google()
         mavenCentral()
     }
 
     dependencies {
-        classpath("org.jetbrains.compose:compose-gradle-plugin:1.2.0")
-        classpath("org.jetbrains.kotlin:kotlin-gradle-plugin:1.7.10")
+        classpath("org.jetbrains.compose:compose-gradle-plugin:1.2.1")
+        classpath("org.jetbrains.kotlin:kotlin-gradle-plugin:1.7.20")
         classpath("com.android.tools.build:gradle:7.2.2")
-        classpath("org.jetbrains.kotlin:kotlin-serialization:1.7.10")
+        classpath("org.jetbrains.kotlin:kotlin-serialization:1.7.20")
```

Comment:
I would not raise the versions in this PR. I've had some issues when updating, so stuck with these older versions.

Reply:
This fixes warnings and (I think) an error I was getting initially with the android setup.

I'm guessing you probably copied over the templates for KMP which had outdated versions.. i tested running both desktop and the android app with this PR and these changes.